### DISCUSSION TITLE
Can't install due to outdated version?

### DISCUSSION THREAD

**Discussion by robinkoehler** [croneter/PlexKodiConnect#1588] [Q&A]
Hi all, bit of a weird one. I am pretty sure the problem is on my end, but still wanted to ask for your advice. 

My Kodi 19.1 Matrix crashed the other day, and after force quitting and reopening, PKC kept asking me to verify my Plex credentials as if it's the first sign-in. No matter how many times I verified, it still would not sync up again. 

I uninstalled PKC, I re-installed Kodi matrix 19.1 on top of the old installation.

I am trying to reinstall PKC Matrix Stable just like I have done many times before with the official method. 

Installing via "zip" file and pulling it from the repository installs version 3.0.0

Now here's the kicker: If I try to install the helper files via repository, it says the minimum version has to be 3.0.1 and that it can't be installed :-/

I then got desperate and tried to install manually via zip file, and I get the error that xmbc.python is not the latest version and that the helper files can't be installed manually?!?!

Is this a know issue, is there a workaround, or is my installation just screwed up and I have to nuke everything and start over?

Thank you kindly for your time and for any replies!

Best wishes, 

Robin

**Comment by croneter** [comment]
1) uninstall PKC completely, following the second post here: https://github.com/croneter/PlexKodiConnect/issues/1359
2) install from scratch following https://github.com/croneter/PlexKodiConnect/wiki/Installation

**Reply by robinkoehler** [reply]
Thanks so much croneter for replying to this! What an amazing tool you wrote here by the way, it was the only thing making the plex library usable for me.

I have followed the instructions in the post you've linked and checked both folders for any orphans. There was nothing left over I could delete.

I have followed the official installation instructions (like many times previously without problems), and I'm still getting version 3.0.0 after installing the Matrix Stable file, and then the "plexkodiconnect.movies version 3.0.1 is not satisfied" error afterwards if I try to install the helper files from the repository. 

I appreciate your help and just wanted to double check that I'm not doing something too stupid, or that I'm missing the obvious solution. 

As I've said, I'm pretty sure it is some particularity with my corrupted Kodi installation that I have not been able to resolve. 

Many thanks again for providing this amazing program! Will report if and when I find a solution. 

Best wishes, 

Robin

**Reply by croneter** [reply]
Do NOT install the helper files directly, but simply 1) the PKC repository 2) PlexKodiConnect itself. It's all there, don't know what your issue is. If that does not help, share a DEBUG log file: https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Reply by robinkoehler** [reply]
Hi Croneter, 

Thank you again for another response. I am first installing the repository via zip file (I had added the url as a source), and as a second step am installing PKC from the new repository that gets added after step one. 

This is the method outlined in the wiki that I had followed successfully several times. Following the exact same steps gives me version 3.0.0 and then the version error when I want to install PKC in the last step.

I am not installing the helper files manually. Had tried that previously in desperation and got knocked back with a xbmc.python version error. 

I have attached the log as you suggested (and scanned for "token", but it doesn't seem to have my Plex tokens in there), it gets interesting on 2021-08-16 20:18:06.075 with these lines:

2021-08-16 20:18:06.075 T:12232   DEBUG <general>: CAddonInstaller: installing 'plugin.video.plexkodiconnect' version '3.4.1' from repository 'repository.plexkodiconnect'
2021-08-16 20:18:06.076 T:10048   DEBUG <general>: CAddonInstallJob[plugin.video.plexkodiconnect]: requires plugin.video.plexkodiconnect.movies version 3.0.1 which is not available
2021-08-16 20:18:06.076 T:10048   ERROR <general>: CAddonInstallJob[plugin.video.plexkodiconnect]: The dependency on plugin.video.plexkodiconnect.movies version 3.0.1 could not be satisfied.
2021-08-16 20:18:06.109 T:12232   DEBUG <general>: CGUIMediaWindow::GetDirectory (addons://repository.plexkodiconnect/xbmc.addon.video)
2021-08-16 20:18:06.109 T:12232   DEBUG <general>:   ParentPath = [addons://repository.plexkodiconnect/xbmc.addon.video]

[kodi.log](https://github.com/croneter/PlexKodiConnect/files/6991580/kodi.log)

**Reply by croneter** [reply]
I have absolutely no idea why you're having this issue. Looks like Kodi does not grab the latest ZIP-files 😞 . Do this:
* Install the stable repository just like you did
* Install these 2 add-ons from ZIP (download and then access them from within Kodi, install as ZIP):
    * https://github.com/croneter/plugin.video.plexkodiconnect.tvshows/archive/refs/tags/3.0.1.zip 
    * https://github.com/croneter/plugin.video.plexkodiconnect.movies/archive/refs/tags/3.0.1.zip
* Install PlexKodiConnect from the PKC repo (should work now)

**Reply by robinkoehler** [reply]
We have liftoff!!! That did it :-)  First try, no problem. 

Sorry to take up so much of your time with such a weird error. I believe I also used the wrong terms in my initial post regarding helper files and the actual PKC. 

The installation completed without issues. I got a bit worried because my folders didn't work, but after restarting Kodi, ALL of my stuff was exactly the way I had it configured before, with all favourites and links etc.

Absolute legend. Enjoy a few coffees :-)

**Reply by croneter** [reply]
Wow, thanks for the coffees!! ❤️
